openpype/hosts/maya/plugins/publish/convert_gltf_shader.py

```diff
@@ -32,10 +32,12 @@ def process(self, instance):
                              glsl_shadingGrp + ".surfaceShader")
 
             # load the maya2gltf shader
-            maya_dir = os.getenv("MAYA_APP_DIR")
-            if not maya_dir:
-                raise RuntimeError("MAYA_APP_DIR not found")
-            ogsfx_path = maya_dir + "/maya2glTF/PBR/shaders/"
+            maya_publish = instance.context.data["project_settings"]["maya"]["publish"] # noqa
+
+            ogsfx_path = maya_publish["ConvertGLSLShader"]["ogsfx_path"]
+
+            if not os.path.exists(ogsfx_path):
+                raise RuntimeError("the ogsfx file not found")
 
             ogsfx = ogsfx_path + "glTF_PBR.ogsfx"
```

Comment:
Wouldn't it make more sense if the `ogsfx_path` in settings would point to the actual file itself and NOT the folder?